src/contexts/Maci.tsx

```diff
@@ -44,6 +44,8 @@ export const MaciProvider: React.FC<MaciProviderProps> = ({ children }: MaciProv
     { enabled: Boolean(maciPubKey && config.maciSubgraphUrl) },
   );
 
+  const poll = api.maci.poll.useQuery({ pollId: "0" }, { enabled: Boolean(config.maciSubgraphUrl) });
```

Comment:
Why is it always "0" poll?

Reply:
Right but the point of this feature is to be able to get this data without having a wallet connected (for instance if a user just wanted to view the results)

Reply:
I should probably add that if there is a wallet connected it could default to calling `getPoll` - or eventually are we going to get rid of all of these functions in favour of subgraph only?

Reply:
What's about using provider without connected wallet to get latest poll id?

Reply:
Yes that's an option, we'd need to set an rpc for it to work, which is doable. what about using the maci collection in the subgraph (it has a field latestPoll) and fetch the poll using that

Reply:
I think we can fetch it using a simple way: just sort poll by id or date created and limit it to one element.
In this case, you don't need to fetch all the polls and you don't need to know latest poll id.
This can be done with only maci-rpgf and no need to change anything in maci-subgraph.

Reply:
oh nice yea I'll do this